eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -0,0 +1,240 @@
+
+              - pwsh: |
+                  $(Build.SourcesDirectory)/eng/versioning/scan_for_unreleased_dependencies.ps1 ${{artifact.groupId}} ${{artifact.name}} $(Build.SourcesDirectory)/sdk/${{parameters.ServiceDirectory}}
+                displayName: "Verify no unreleased dependencies"
+
+
+          - deployment: TagRepository
+            displayName: "Create release tag"
+            condition: and(succeeded(), ne(variables['Skip.TagRepository'], 'true'))
+            environment: github
+            timeoutInMinutes: 5
+            dependsOn: VerifyReleaseVersion
+
+            pool:
+              vmImage: vs2017-win2016
+
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - checkout: self
+                    - pwsh: |
+                        Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                        New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
+                        Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}-[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
+                        Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
+                      workingDirectory: $(Pipeline.Workspace)
+                      displayName: Stage artifacts
+                      timeoutInMinutes: 5
+                    - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
+                      parameters:
+                        ArtifactLocation: $(Pipeline.Workspace)/${{artifact.safeName}}
+                        PackageRepository: Maven
+                        ReleaseSha: $(Build.SourceVersion)
+                        RepoId: Azure/azure-sdk-for-java
+
+          - ${{if ne(artifact.options.skipPublishPackage, 'true')}}:
+            - deployment: PublishPackage
+              displayName: "Publish to Maven Central"
+              condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
+              environment: maven
+              dependsOn: TagRepository
+
+              pool:
+                vmImage: windows-2019
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: azure-sdk-build-tools
+                        path: azure-sdk-build-tools
+                      - template: tools/gpg/gpg.yml@azure-sdk-build-tools
+                      - template: /eng/pipelines/templates/steps/java-publishing.yml
+                        parameters:
+                          ArtifactID: ${{artifact.name}}
+                          GroupID: ${{artifact.groupId}}
+                          ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                          Target: MavenCentral
+                          BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
```

Comment:
I assume BuildToolsPath is no longer needed.

Reply:
It's still used to reference gpg.exe

Reply:
Nevermind it is still used for the gpg tool. Perhaps to make that clear we should update it to that.